CHANGELOG.md

```diff
@@ -96,6 +97,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -  [Single sign-on for Microsoft Teams apps](https://microsoft.github.io/BotFramework-WebChat/19.c.single-sign-on-for-teams-apps/), by [@compulim](https://github.com/compulim) in [#2196](https://github.com/microsoft/BotFramework-WebChat/pull/2196)
 -  [Customize Web Chat with Reaction Buttons](https://microsoft.github.io/BotFramework-WebChat/09.customization-reaction-buttons/). Updated reaction handlers to send `messageReaction` activities, by [@tdurnford](https://github.com/tdurnford) in [#2239](https://github.com/microsoft/BotFramework-WebChat/pull/2239)
 
+## [4.5.3] - 2019-16-07
```

Comment:
I don't think we're going to add this to the minor patch, so could you delete line 102? Thank you! Please also sign your SLA so that we can accept your pull request.

Reply:
Sorry about that! 
I have removed that section in the CHNAGELOG.md file and signed the SLA.

Thanks again for all your help and patience!